client/app/scripts/superdesk-authoring/metadata/metadata.js

```diff
@@ -3,8 +3,14 @@
 
 'use strict';
 
-MetadataCtrl.$inject = ['$scope', 'desks', 'metadata', '$filter', 'privileges', 'datetimeHelper'];
-function MetadataCtrl($scope, desks, metadata, $filter, privileges, datetimeHelper) {
+MetadataCtrl.$inject = [
+    '$scope', 'desks', 'metadata', '$filter', 'privileges', 'datetimeHelper',
+    'preferencesService'
+];
+function MetadataCtrl(
+    $scope, desks, metadata, $filter, privileges, datetimeHelper,
+    preferencesService
+) {
```

Comment:
What would be a good alternative here? The line was a bit too long and started wrapping already.
(the question also applies to other similar cases as well where lines are _definitely_ too long)